src/step_definitions/importSteps.js

```diff
@@ -15,3 +15,16 @@ When(
     await importPage.setValue("@listDescriptionField", args1);
   }
 );
+
+Then(
+  /^user can see "([^"]*)" on "([^"]*)" of Preview Table$/,
```

Comment:
It would be more clear if the step read 
user can see **Synonyms** "([^"]*)" **on line** "([^"]*)" of Preview Table$

Reply:
Done

Reply:
look good